### DISCUSSION TITLE
Running on a different port

### DISCUSSION THREAD

**Discussion by bcnx** [reconmap/reconmap#95] [Q&A] [answered]
Hi,

docker-compose up -d failed because I had port 8080 in use by Burp. Can I change the port before running docker-compose?

Cheers,

BC

**Comment by santiagolizardo** [comment, marked as answer]
Hi @bcnx ; Yes it is possible. You just need to change the ports in the [docker compose file](https://github.com/reconmap/reconmap/blob/master/docker-compose.yml) and then adjust the [config file for the web client](https://github.com/reconmap/reconmap/blob/master/environment.js)  accordingly. 
Hope that helps.
